roles/KubevirtCommonTemplatesBundle/tasks/main.yml

```diff
@@ -118,3 +118,17 @@
     status:
       observedVersion: "{{ operator_version }}"
   when: "{{ available==true }}"
+
+- name: Create namespace for golden images
+  k8s:
+    kind: Namespace
+    name: "{{ base_image_namespace }}"
+    state: present
+
+- name: Create RBAC for Golden Images
+  k8s:
+    state: present
+    definition: "{{ item | from_yaml }}"
+    merge_type: ['merge', 'json']
+  with_items: "{{ lookup('template', 'kubevirt-base-image-roles.yaml.j2').split('\n---\n') | select('search', '(^|\n)[^#]') | list }}"
```

Comment:
you don't have to parse the file here and create it one by one. Just load the file and create all at once (see here https://github.com/kubevirt/kubevirt-ssp-operator/blob/master/roles/KubevirtCommonTemplatesBundle/tasks/main.yml#L13)